workers/pipeline/update_pipeline.go

```diff
@@ -79,9 +85,45 @@ func updatePipeline(p *gaia.Pipeline) error {
 		// Install gem forcefully.
 		cmd := exec.Command(path, "install", "-f", pipelineCopyPath)
 		if out, err := cmd.CombinedOutput(); err != nil {
-			gaia.Cfg.Logger.Error("error", string(out[:]))
+			return fmt.Errorf("cannot install ruby gem: %s", string(out[:]))
+		}
+	case gaia.PTypeNodeJS:
+		// Find tar binary in path
+		path, err := exec.LookPath(tarName)
+		if err != nil {
+			return err
+		}
+
+		// Delete old folders if exist
+		tmpFolder := filepath.Join(gaia.Cfg.HomePath, gaia.TmpFolder, gaia.TmpNodeJSFolder, p.Name)
+		os.RemoveAll(tmpFolder)
+
+		// Recreate the temp folder
+		if err := os.MkdirAll(tmpFolder, 0700); err != nil {
+			return err
+		}
+
+		// Unpack it
+		cmd := exec.Command(path, "-xzvf", p.ExecPath, "-C", tmpFolder)
```

Comment:
I'm wondering if we really need the `-v` flag? I don't think we really need this to be verbose, don't we?

Reply:
If the command fails, the output is displayed in the logs. This could be helpful.